apacheconfig/wloader.py

```diff
@@ -31,19 +31,135 @@ def dump(self):
     def typestring(self):
         """Returns object typestring as defined by the apacheconfig parser."""
 
-    @abc.abstractproperty
-    def whitespace(self):
-        """Returns preceding or trailing whitespace for this node as a string.
+
+class ListNode(AbstractASTNode):
+    """Creates object representing an ordered list of LeafASTNodes.
+
+    Each BlockNode contains a ListNode, and every configuration file's
```

Comment:
May be some sort of ASCII diagram of parsed config tree explaining the relationship between these classes somewhere would be easier to grasp? Including recursion wherever it can occur.